__tests__/articles.test.js

```diff
@@ -0,0 +1,55 @@
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data");
+const request = require("supertest");
+const articlesData = require("../db/data/test-data/articles");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/articles", () => {
+  describe("GET /api/articles/:article_id", () => {
```

Comment:
We should only have one describe block here- line 21 can be deleted as you're only describing the endpoint on line 22 👍